host/windows/syscall.c

```diff
@@ -3124,3 +3124,28 @@ int oe_syscall_nanosleep_ocall(struct oe_timespec* req, struct oe_timespec* rem)
     _set_errno(0);
     return 0;
 }
+
+/*
+**==============================================================================
+**
+** clock_nanosleep():
+**
+**==============================================================================
+*/
+
+int oe_syscall_clock_nanosleep_ocall(
+    oe_clockid_t clockid,
+    int flag,
+    struct oe_timespec* req,
+    struct oe_timespec* rem)
+{
+    /*
+     * Since clock selection and interval specification are linux specific
+     * features, we ignore clockid and flag input parameters and call
+     * oe_syscall_nanosleep_ocall.
```

Comment:
I think ignoring arguments iwill lead to bugs.   This should instead explicitly check for what we support and fail anything we don't support, rather than silently doing something other than what the caller requested.

Reply:
Thank you for pointing this out!